databricks/koalas/indexes.py

```diff
@@ -831,6 +831,64 @@ def name(self) -> str:
     def name(self, name: str) -> None:
         raise PandasNotImplementedError(class_name='pd.MultiIndex', property_name='name')
 
+    def swaplevel(self, i=-2, j=-1):
+        """
+        Swap level i with level j.
+        Calling this method does not change the ordering of the values.
+
+        Parameters
+        ----------
+        i : int, str, default -2
+            First level of index to be swapped. Can pass level name as string.
+            Type of parameters can be mixed.
+        j : int, str, default -1
+            Second level of index to be swapped. Can pass level name as string.
+            Type of parameters can be mixed.
+
+        Returns
+        -------
+        MultiIndex
+            A new MultiIndex.
+
+        Examples
+        --------
+        >>> midx = ks.MultiIndex.from_arrays([['a', 'b'], [1, 2]], names = ['word', 'number'])
+        >>> midx # doctest: +SKIP
+        MultiIndex([('a', 1),
+                    ('b', 2)],
+                   names=['word', 'number'])
+
+        >>> midx.swaplevel(0, 1) # doctest: +SKIP
+        MultiIndex([(1, 'a'),
+                    (2, 'b')],
+                   names=['number', 'word'])
+
+        >>> midx = ks.MultiIndex.from_arrays([['a', 'b'], [1, 2]], names = ['word', 'number'])
+        >>> midx.swaplevel('number', 'word') # doctest: +SKIP
+        MultiIndex([(1, 'a'),
+                    (2, 'b')],
+                   names=['number', 'word'])
+        """
+        index_columns = self._internal.index_columns
+        sdf = self._internal._sdf
+        index_map = self._internal.index_map
+        if any(map(lambda x: x is None, self._internal.index_names)):
```

Comment:
let's consider about 

`names=['word', None]`

in pandas,

```python
>>> midx = pd.MultiIndex.from_arrays([['a', 'b'], [1, 2]], names = ['word', None])
>>> midx
MultiIndex([('a', 1),
            ('b', 2)],
           names=['word', None])

>>> midx.swaplevel(0, 1)
MultiIndex([(1, 'a'),
            (2, 'b')],
           names=[None, 'word'])
```

whereas koalas,

```python
>>> midx = ks.MultiIndex.from_arrays([['a', 'b'], [1, 2]], names = ['word', None])
>>> midx
MultiIndex([('a', 1),
            ('b', 2)],
           names=['word', None])
>>>
>>> midx.swaplevel(0, 1)
MultiIndex([(1, 'a'),
            (2, 'b')],
           names=['word', None])
```

it seems names not changed unlike pandas.

Reply:
Hello @itholic i.Can you help the problem?
Why the level changes when function called without signing?
```
midx = ks.MultiIndex.from_arrays([['a', 'b'], [1, 2]], names = ['word', 'number'])
midx.swaplevel(0, 1)
MultiIndex([(1, 'a'),
               (2, 'b')],
               names=['number', 'word'])
midx._internal.index_map
['number', 'word']
```
The level order changes permanently without signing something or set `inplace=True` parameter. Thanks